src/Avalonia.Controls.DataGrid/DataGridTextColumn.cs

```diff
@@ -36,93 +31,91 @@ public DataGridTextColumn()
         /// <summary>
         /// Identifies the FontFamily dependency property.
         /// </summary>
-        public static readonly StyledProperty<string> FontFamilyProperty =
```

Comment:
This was a bug and you fixed it. Not sure why it was defined as a string. This worked because of build in conversion from string to FontFamily